src/registrar/utility/csv_export.py

```diff
@@ -215,7 +218,7 @@ def export_data_type_to_csv(csv_file):
     ]
     filter_condition = {
         "domain__state__in": [
-            Domain.State.READY,
+            Domain.State.UNKNOWN,
```

Comment:
Don't forget to remove this when you merge 😆

Reply:
DUDE!